packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx

```diff
@@ -26,13 +29,9 @@ export type LabeledInputProps = {
    */
   svgIcon?: JSX.Element;
   /**
-   * Custom CSS class name for the input element.
+   * Pass props to wrapper element.
    */
-  inputClassName?: string;
-  /**
-   * Custom CSS Style for the input element.
-   */
-  inputStyle?: React.CSSProperties;
```

Comment:
make sure to document these breaking changes

Reply:
Documented in changeset